packages/payment-processor/package.json

```diff
@@ -44,7 +44,8 @@
     "@requestnetwork/smart-contracts": "0.28.0",
     "@requestnetwork/types": "0.35.0",
     "@requestnetwork/utils": "0.35.0",
-    "ethers": "5.5.1",
```

Comment:
Was the `ethers.js` update needed? Because this would mean updating the other packages that depend on it too.

Reply:
I've rollbacked to initial version of ethers.js and it shows no updated version is needed.